src/irmin-pack/unix/s.ml

```diff
@@ -16,20 +16,6 @@
 
 open! Import
 
-exception RO_not_allowed
-
-module type Checkable = sig
-  type 'a t
-  type hash
-
-  val integrity_check :
-    offset:int63 ->
-    length:int ->
-    hash ->
-    _ t ->
-    (unit, [ `Wrong_hash | `Absent_value ]) result
-end
-
 (** [Irmin-pack]-specific extensions to the [Store] module type. *)
 module type Specifics = sig
```

Comment:
The `Specifics` level of abstraction exists solely for irmin_pack-unix. Have you tried to remove it and inline it's content in `module type S` below?

Reply:
+1 to this idea

Reply:
done, thanks